https://lwn.net/Articles/923969/
https://sources.debian.org/data/main/e/e2fsprogs/1.47.0-2/debian/patches/disable-metadata_csum_seed-and-orphan_file-by-default

Description: Disable metadata_csum_seed and orphan_file by default for Debian
 .
 There are a lot of debian packages that assume that mkfs.ext4 for
 Debian N will work on Debian N-1 (and sometimes N-2, N-3, etc.).  This
 is a bad assumption, but it's too late to fix this for Debian
 Bookworm.
 .
 So revert making these feature a default for now.  We will enable it
 again post Bookworm.
Author: Theodore Ts'o <tytso@mit.edu>
Origin: vendor
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1031622
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1030939
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,7 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,metadata_csum_seed,64bit,dir_nlink,extra_isize,orphan_file
+		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
 	}
 	small = {
 		blocksize = 1024
-- 
2.31.0
